Vision/Vision.bs

```diff
@@ -180,8 +180,8 @@ Markup Shorthands: markdown yes
 	<li id=incub>
 		**Incubation**: The Web will continue to expand
 		in user base, global reach, and technical breadth. 
-		We encourage and facilitate incubation in new areas, 
-		collaborating on innovations across our community.
+		We encourage and facilitate careful and responsible exploration of new areas and ideas, 
+		and intentional collaboration on innovations across our community.
```

Comment:
If your goal is to say something about growing platform capabilities, say that.

```suggestion
		**New Capabilities**: The Web will continue to expand
		in user base, global reach, and technical capabilities. 
		We encourage and facilitate careful and responsible exploration of new areas and ideas, 
		and intentional collaboration on innovations across our community.
```

"incubation" is for eggs.